pkg/operations/kubernetesupgrade/upgrader.go

```diff
@@ -263,6 +265,7 @@ func (ku *Upgrader) upgradeAgentPools(ctx context.Context) error {
 		} else {
 			upgradeAgentNode.timeout = *ku.stepTimeout
 		}
+		upgradeAgentNode.cordonDrainTimeout = *ku.cordonDrainTimeout
```

Comment:
this is the place I'm suggesting we move the default, same as what stepTimeout is doing just above